cdbot/cogs/cyber.py

```diff
@@ -360,6 +360,72 @@ def __init__(self, bot: Bot):
         await ctx.send(f"There are {len(game_r.members)} that have completed Cyberstart Game. Out of them, "
                        f"{len(true_r.members)} have also completed Essentials and Assess.")
 
+    @command()
+    async def elitecountdown(self, ctx: Context, event_name: str = None, event_level: str = None):
+        """
+        Gets the time until elite, and shows in an embed
+        """
```

Comment:
The syntax for this command is never displayed. There should be a response to a command like this explaining how to use the command:
![image](https://user-images.githubusercontent.com/3729048/59565876-5cc97200-9050-11e9-979c-52903543a9fa.png)

Reply:
`:elitecountdown` on its own will list all events